### PR TITLE
add multiple sections for dsp subsystem

### PR DESCRIPTION
**Pull request by Victor-M-Giraldo**

### CODE REVIEW

**Comment by Victor-M-Giraldo** [comment]
I do have sources for most of these. Will add tomorrow. Will also update slides tomorrow.

File: subsys-dsp.typ [hunk 1]
```diff
@@ -2,6 +2,42 @@
...
+
+- Error Handling: The DSP will use an ARQ error-control strategy, or automatic repeat request. This process is described in detail in @Figure::dsp-arq.
+
+== Signal Integrity
+
+The DSP will employ a modulation technique called phase-shift keying. Phase-shift keying helps maintain signal clarity over vast distances.
+
+== Security
+
+The DSP will implement a security framework developed by NASA called the Genomic and Proteomic Information Security Protocol. It utilizes advanced encryption and authentication techniques derived from genomic and proteomic data.
```
Comment (dyc3) [inline subsys-dsp.typ:35]: Interesting

File: subsys-dsp.typ [hunk 2]
```diff
@@ -2,6 +2,42 @@
 
 The Data Streaming Protocol (DSP) facilitates the streaming of metric data from probes and the issuance of commands to probes. It is optimized for minimal connection setup time and robustness to packet loss and signal degradation.
 
+== Architecture
+
+The Data Streaming Protocol (DSP) is a specialized communication framework designed to facilitate real-time data streaming from space probes across vast distances. It comprises a dual-module setup with client-side components on the probes and server-side components at Earth stations.
```
Comment (dyc3) [inline subsys-dsp.typ:7]: Should have caught this in the previous review, but the DSP needs to be able to issue commands to probes as well.

**Review by dyc3 — COMMENTED** [review]
I wanna hold off on merging this until there's citations.

**Comment by Victor-M-Giraldo** [comment]
@dyc3 I'll address all that today. Do you think we can add an auto format to the .typ files so lines wrap after a certain point? As is it's kind of annoying to read.

**Comment by dyc3** [comment]
Turn on word wrap in your editor and/or github

**Comment by Victor-M-Giraldo** [comment]
Oh neat I didn't know that was a thing. Thanks.

File: subsys-dsp.typ [hunk 3]
```diff
@@ -2,6 +2,42 @@
...
+
+- Decoding and Decompression: At Earth stations, the server module decodes the incoming data streams. In this step, the compression algorithm is reverse to restore the original data. The metadata is also checked to correctly route the data for analysis.
+
+== Communication and Transmission
+
+- Transmission Control: The DSP will use the Additive Increase/Multiplicative Decrease (AIMD) algorithm. Each node starts with a certain access probability which is periodically adjusted: it increases by a small fixed amount if conditions are favorable (no congestion detected), and decreases multiplicatively when congestion occurs @li2013aimdmac.
+
+- Error Handling: The DSP will use an ARQ error-control strategy, or automatic repeat request. This process is described in detail in @Figure::dsp-arq.
```
Comment (dyc3) [inline subsys-dsp.typ:27]: The citations render as `[1]` so this sentence will look weird.

**Review by dyc3 — CHANGES_REQUESTED** [review]
nit: place the citations after the period

Otherwise this looks good, as long as issuing commands gets addressed in another PR.

**Review by dyc3 — APPROVED** [review]

**Pull request merged**